### PR TITLE
Campaigns splash screens

### LINKED ISSUE #9140: Campaigns splash screen
For first time campaigns users and customers that don't have a tier with access to campaigns yet, show an informative splash screen that describes the campaigns feature.

### LINKED ISSUE #9555: Remove feature flag for campaigns in navbar
Only show the splash page, campaigns is still going to be _Beta_. On dotcom, we are showing different content, but also _some_ content.

### PR DESCRIPTION
**Pull request by eseliger**
Closes #9140 
Closes #9555

### CODE REVIEW

**Comment by christinaforney** [comment]
@eseliger - I started doing copy edits in a review, but ended up needing to play with the formatting. I pushed a commit with my suggestions, please make sure I didn't do anything silly 😄 

Updated screenshots:

Non-admin (with campaigns enabled on the instance):
image

Admin:
image

Sourcegraph.com user:
image

 cc @adaaaam - after seeing this implemented, it felt very disconnected from the rest of Sourcegraph and felt a bit too much like our About site. Instead, we chose to keep it simple, point at the about site Campaigns content, and focus on what the user needs to do. Looking forward to your feedback on this update!

**Comment by eseliger** [comment]
@christinaforney looks good :)

FYI, in this PR I removed the addition of pings, I will look into it after I finished up another important PR and if I find a reasonable approach to it, we can cherry-pick it

File: web/src/enterprise/campaigns/global/marketing/CampaignsMarketing.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+                    for more information.
+                </em>
+            </p>
+            <h2 className="mb-6">Make large-scale code changes across all your repositories and code hosts.</h2>
+            <p className="mt-3">
+                <a href="https://about.sourcegraph.com/product/code-change-management" rel="noopener">
+                    Learn how
+                </a>{' '}
+                campaigns can help you remove legacy code, fix critical security issues, and pay down tech debt. See it
+                in action to run <code>go mod tidy</code> over all Go repositories:
```
Comment (mrnugget) [inline web/src/enterprise/campaigns/global/marketing/CampaignsMarketing.tsx:26]:
```suggestion
                in action to run <code>gofmt</code> over all Go repositories:
```

The teaser image of the video is wrong.

File: web/src/enterprise/campaigns/global/marketing/CampaignsSiteAdminMarketingPage.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+                        Optional: enable read-only access for all users.
+                        <div className="alert alert-warning mt-3">
+                            <strong>WARNING:</strong> Repository permissions are NOT enforced if you enable read-only
+                            views on campaigns. Any logged in user will be able to see all code changes associated with
+                            a campaign. Therefore, read-only access is recommended for instances without repository
+                            permissions configured.
+                        </div>
+                    </li>
+                    <li>
+                        <a href="https://docs.sourcegraph.com/user/campaigns#creating-campaigns">
```
Comment (mrnugget) [inline web/src/enterprise/campaigns/global/marketing/CampaignsSiteAdminMarketingPage.tsx:50]:
```suggestion
                        <a href="https://docs.sourcegraph.com/user/campaigns">
```

The new docs will have a "Are you a new Campaigns user section?" so I think we should link them to the docs entry point.

File: web/src/enterprise/campaigns/global/marketing/CampaignsUserMarketingPage.tsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+                    At this time, creating and managing campaigns is only available to Sourcegraph admins. What can you
+                    do?
+                </p>
+                <div className="row">
+                    <ol>
+                        <li>Let your Sourcegraph admin know you're interested in using Campaigns for your team.</li>
+                        {enableReadAccess && (
+                            <li>
+                                Ask your Sourcegraph admin for{' '}
+                                <a href="https://docs.sourcegraph.com/user/campaigns#configuration" rel="noopener">
```
Comment (mrnugget) [inline web/src/enterprise/campaigns/global/marketing/CampaignsUserMarketingPage.tsx:26]:
```suggestion
                                <a href="https://docs.sourcegraph.com/user/campaigns" rel="noopener">
```

Everywhere else we link to the docs entry page, so I think it makes sense to be consistent

**Pull request merged**